--- jquery.js	2024-11-27 21:01:02.597526047 +1100
+++ jquery-corrected-2.2.4.js	2024-11-28 23:17:27.202276753 +1100
@@ -1,4 +1,4 @@
 /*!
- * jQuery JavaScript Library v2.2.4
+ * jQuery JavaScript Library v2.2.4C
  * http://jquery.com/
  *
@@ -544,5 +544,5 @@
 }
 var Sizzle =
-/*
+/*!
  * Sizzle CSS Selector Engine v2.2.1
  * http://sizzlejs.com/
@@ -3745,7 +3745,9 @@
 	},
 	get: function( owner, key ) {
+		try {
 		return key === undefined ?
 			this.cache( owner ) :
 			owner[ this.expando ] && owner[ this.expando ][ key ];
+		} catch (e) {}
 	},
 	access: function( owner, key, value ) {
